man/systemd.network.xml

```diff
@@ -1926,16 +1926,14 @@
         <term><varname>EmitDNS=</varname></term>
         <term><varname>DNS=</varname></term>
 
-        <listitem><para><varname>DNS=</varname> specifies a list of recursive
-        DNS server IPv6 addresses that distributed via Router Advertisement
-        messages when <varname>EmitDNS=</varname> is true. If <varname>DNS=
-        </varname> is empty, DNS servers are read from the
-        <literal>[Network]</literal> section. If the
-        <literal>[Network]</literal> section does not contain any DNS servers
-        either, DNS servers from the uplink with the highest priority default
-        route are used. When <varname>EmitDNS=</varname> is false, no DNS server
-        information is sent in Router Advertisement messages.
-        <varname>EmitDNS=</varname> defaults to true.
+        <listitem><para><varname>DNS=</varname> specifies a list of recursive DNS server IPv6 addresses
+        that distributed via Router Advertisement messages when <varname>EmitDNS=</varname> is true.
+        <varname>DNS=</varname> also takes special value <literal>linklocal</literal>, in that case the
+        IPv6 link local address is distributed. If <varname>DNS=</varname> is empty, DNS servers are
```

Comment:
I don't think we should use a valid DNS name here. Can we change this to "_link_local" please? This has the advantage that it is obviously a special name, not a DNS hostname, and we avoid any conflicts. It'll also be more extensible if we want to add more special cases in the future.